lib/maths/unittest/CTimeSeriesDecompositionTest.cc

```diff
@@ -1872,11 +1873,11 @@ void CTimeSeriesDecompositionTest::testYearly() {
         }
     }
 
-    std::ofstream file;
-    file.open("results.m");
-    TDoubleVec f;
-    TTimeVec times;
-    TDoubleVec values;
+    //std::ofstream file;
```

Comment:
Should these be removed?

Reply:
These are useful to have in if there are changes in results. You can uncomment these and look at the modelling we do.